docs/standard/microservices-architecture/multi-container-microservice-net-applications/data-driven-crud-microservice.md

```diff
@@ -43,7 +43,7 @@ To implement a simple CRUD microservice using .NET Core and Visual Studio, you s
 
 **Figure 8-6**. Creating an ASP.NET Core Web API project in Visual Studio
 
-After creating the project, you can implement your MVC controllers as you would in any other Web API project, using the Entity Framework API or other API. In the eShopOnContainers.Catalog.API project, you can see that the main dependencies for that microservice are just ASP.NET Core itself, Entity Framework, and Swashbuckle, as shown in Figure 8-7.
+After creating the project, you can implement your MVC controllers as you would in any other Web API project, using the Entity Framework API or other API. In a new Web API project, you can see that the only dependency you have in that microservice is on ASP.NET Core itself. Internally, within the `Microsoft.AspNetCore.All` dependency, it is referencing to Entity Framework and many other .NET Core Nuget packages, as shown in Figure 8-7.
```

Comment:
delete "to" before "to Entity Framework"

Reply:
👍